doc/tutorial/tutorial.md

```diff
@@ -135,30 +130,31 @@ should use the order that makes programs easiest to read.
 
 ### Compiling the "Hello, world!" program
 
-Run `stack test --ta '-p playpen'` to compile this program.
-(Unfortunately the Rust compiler is quite slow, so this may take a few
-minutes).
-
-Alternatively, run the DDlog compiler directly to produce the Rust program
+Compiling a DDlog program consists of two steps.
+First, run the DDlog compiler to generate the Rust program:
 ```
-ddlog -i playpen.dl -L../../lib
+ddlog -i playpen.dl -L <ddlog/lib>
```

Comment:
We should add an environment variable for this.

Reply:
Yes!